sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -250,16 +254,20 @@ internal class ViewAnnotationManagerImpl(
     val onDrawListener = ViewTreeObserver.OnDrawListener {
       if (viewAnnotation.handleVisibilityAutomatically) {
         val isAndroidViewVisible = (inflatedView.visibility == View.VISIBLE)
-        if ((isAndroidViewVisible && viewAnnotation.isVisible) ||
-          (!isAndroidViewVisible && viewAnnotation.visibility == ViewAnnotationVisibility.INVISIBLE)
+        if (
+          (isAndroidViewVisible && viewAnnotation.isVisible) ||
+          (!isAndroidViewVisible && viewAnnotation.visibility == ViewAnnotationVisibility.INVISIBLE) ||
+          (!isAndroidViewVisible && viewAnnotation.visibility == ViewAnnotationVisibility.VISIBLE_AND_NOT_POSITIONED)
```

Comment:
nit:
```suggestion
          (!isAndroidViewVisible && (viewAnnotation.visibility == ViewAnnotationVisibility.INVISIBLE) || (viewAnnotation.visibility == ViewAnnotationVisibility.VISIBLE_AND_NOT_POSITIONED))
```

Reply:
I found it less readable, actually, thought about it as well